### DISCUSSION TITLE
Python 3.5 support?

### DISCUSSION THREAD

**Discussion by mrx23dot** [python-telegram-bot/python-telegram-bot#2298] [Q&A]
Which older version supports Python 3.5?
Or can you suggest an alternative lib for that?
It might worth mentioning on readme page.

Many embedded devices have only older version of Python ported.

**Comment by Bibo-Joshi** [comment]
Support for Python 3.5 was dropped in v13, as can be found e.g. [here](https://t.me/pythontelegrambotchannel/88).

**Reply by mrx23dot** [reply]
Will it still work in the future with Telegram API or it's mandatory to upgrade?

**Reply by Bibo-Joshi** [reply]
No, it won't, as PTB v13+ is using some python features that are new in py3.6. Also py3.5 has already reached it's end of life anyway.
